iwyu_ast_util.cc

```diff
@@ -1207,7 +1208,7 @@ const Type* RemovePointersAndReferences(const Type* type) {
   return type;
 }
 
-const NamedDecl* TypeToDeclAsWritten(const Type* type) {
+static const NamedDecl* TypeToDeclImpl(const Type* type, bool AsWritten) {
```

Comment:
`AsWritten` is LLVM style, IWYU uses `hacker_case` -- so please make this `as_written`

Reply:
Fixed.